### PR TITLE
SW-5360 Added deliverables to modules

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2612?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2612** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2612?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2607** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2607?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2606** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2606?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/ModuleDetailsCard/index.tsx [hunk 1]
```diff
@@ -152,9 +134,17 @@ const ModuleDetailsCard = ({ projectId, module, showSeeAllModules = false }: Mod
...
-          {!!mockDeliverables.length && (
+          {(!!contents.length || !!deliverables.length) && (
+            <ModuleContentSection>
+              <Typography fontSize={'20px'} lineHeight={'28px'} fontWeight={600}>
+                {strings.THIS_MODULE_CONTAINS}
+              </Typography>
+            </ModuleContentSection>
+          )}
+
+          {!!deliverables.length && (
```
Comment (nickgraz) [inline src/components/ModuleDetailsCard/index.tsx:145]: No need to check the length, `.map` won't return anything if the array is empty

**Review by nickgraz — APPROVED** [review]
Looks good to me, one small nit pick

**Comment by tommylau523** [comment]
### Merge activity

* **May 13, 6:44 PM EDT**: @tommylau523 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2612).

**Pull request merged**